compiler/rustc_middle/src/hir/map/mod.rs

```diff
@@ -114,20 +114,19 @@ impl<'hir> Iterator for ParentOwnerIterator<'hir> {
         if self.current_id == CRATE_HIR_ID {
             return None;
         }
-        loop {
-            // There are nodes that do not have entries, so we need to skip them.
-            let parent_id = self.map.def_key(self.current_id.owner.def_id).parent;
 
```

Comment:
Something is fishy here. At minimum comments need updating

Reply:
Since https://github.com/onur-ozkan/rust/commit/667d5d325f7657e5cdf1cade84e2602c5ebb9a8d, we no longer skip nodes. Removing the comments should be fine I guess?